sunpy/map/tests/test_mapbase.py

```diff
@@ -370,7 +370,10 @@ def test_save(aia171_test_map, generic_map):
     aiamap.save(afilename, filetype='fits', overwrite=True)
     loaded_save = sunpy.map.Map(afilename)
     assert isinstance(loaded_save, sunpy.map.sources.AIAMap)
-    assert loaded_save.meta == aiamap.meta
+    # Compare metadata without considering ordering of keys
+    assert loaded_save.meta.keys() == aiamap.meta.keys()
+    for k in aiamap.meta:
+        assert loaded_save.meta[k] == aiamap.meta[k]
```

Comment:
Can we not compare `.items()` directly instead of a loop?

Reply:
Google seems to suggest not.

Reply:
I'm not sure. I know this is not the nicest approach. I'm happy to do something different here if there is a better option.